documentation/src/main/asciidoc/migration/index.asciidoc

```diff
@@ -22,6 +22,19 @@ If you're looking to migrate from an earlier version,
 you should migrate step-by-step, from one minor version to the next,
 following the migration guide of link:https://hibernate.org/search/documentation/[each version].
 
+[WARNING]
+====
+**To Hibernate Search 5 users**
+
+Be aware that a lot of APIs changed since Hibernate Search, some only because of a package change,
```

Comment:
```suggestion
Be aware that a lot of APIs have changed since Hibernate Search 5, some only because of a package change,
```
looks like version # is missing, or was that on purpose?

Reply:
No you're right, thanks